cinema/serializers.py

```diff
@@ -1 +1,88 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import CinemaHall, Genre, Actor, Movie, MovieSession
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ["name", "rows", "seats_in_row", "capacity"]
```

Comment:
The `capacity` field is a property of the `CinemaHall` model and not a database field. You need to define it explicitly in the `CinemaHallSerializer` using a `SerializerMethodField` or similar approach to include it in the serialized output.